src/fNull.js

```diff
@@ -0,0 +1,46 @@
+import { isNil, curryN } from 'ramda';
+
+import defaultWhen from './defaultWhen';
+import isFunction from './isFunction';
+import isArray from './isArray';
+
+/**
+ * Returns a function which is called with the given arguments. If any of the given arguments are null or undefined,
+ * the corresponding default value for that argument is used instead to call the original function.
+ *
+ * @func fNull
+ * @memberOf RA
+ * @category Function
+ * @sig function, [defaultArgs] -> function -> function(args or defaultArgs)
+ * @param {Function} function to be executed
+ * @param {Array|undefined} default arguments to call the function with
+ * @return {Function} function to be called
+ * @example
+ *
+ * RA.fNull(); // => fn // => undefined
+ * RA.fNull(fn); // => fn // => fn(args) //no default arguments
+ * RA.fNull(fn, [a]) // => fn // => fn(args) first argument with default param as a
```

Comment:
Here we're trying to demonstrate the functionality of the function.

```suggestion
 const addDefaults = RA.fnull((a, b) => a + b, [4, 5])
 
 addDefaults(1, 2); // => 3
 addDefaults(null, 2); // => 6
 addDefaults(2, null); // => 7
 addDefaults(undefined, undefined); // => 9 
```